pcap_linux.go

```diff
@@ -108,15 +110,40 @@ func (h *Handle) ReadPacketData() (data []byte, ci gopacket.CaptureInfo, err err
 
 func (h *Handle) readPacketDataSyscall() (data []byte, ci gopacket.CaptureInfo, err error) {
 	b := make([]byte, h.snaplen)
-	read, _, err := syscall.Recvfrom(h.fd, b, 0)
+	oob := make([]byte, syscall.CmsgSpace(tpacketAuxdataSize))
+	n, _, _, _, err := syscall.Recvmsg(h.fd, b, oob, 0)
```

Comment:
So this is telling it to receive both the regular (non-control) data and the out of band control information? With the control data going to `b` (as before) and the control going to `oob`?

Reply:
correct, `b` is the standard packet capture data, and `oob` is the cmsg containing `PACKET_AUXDATA` in this case.